internal/handler/auth_handlers/register_handler.go

```diff
@@ -0,0 +1,82 @@
+package auth_handlers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/jackc/pgx/v5"
+	dbutils "github.com/yunusemreayhan/goAuthMicroService/db"
+	"github.com/yunusemreayhan/goAuthMicroService/db/sqlc"
+	"github.com/yunusemreayhan/goAuthMicroService/models"
+	"github.com/yunusemreayhan/goAuthMicroService/restapi/operations/auth"
+)
+
+// curl -X POST -H "Content-Type: application/json" -d '{"person_name":"test4", "email":"test4@test", "password":"test4"}' localhost:3000/api/register
+// curl -X POST -H "Content-Type: application/json" -d '{"person_name":"test4", "email":"test4@test", "password":"test4"}' localhost/api/register
+
+type RegisterHandler struct {
+	middleware.Responder
+	req auth.RegisterParams
+}
+
+func (l *RegisterHandler) BindRequest(req auth.RegisterParams) {
+	l.req = req
+}
+
+func (l *RegisterHandler) WriteResponse(rw http.ResponseWriter, pr runtime.Producer) {
+	ctx := context.Background()
+
+	con, err := pgx.Connect(ctx, dbutils.GetSQLDSN())
```

Comment:
There will be a lot of connections to DB in case of multiple requests on a high load.